mapadroid/cache/__init__.py

```diff
@@ -0,0 +1,21 @@
+from mapadroid.cache.noopcache import NoopCache
+from mapadroid.utils.logging import get_logger, LoggerEnums
+
+logger = get_logger(LoggerEnums.system)
+
+
+def get_cache(args):
+    cache = NoopCache()
+    if args.enable_cache:
+        try:
+            import redis
```

Comment:
This import should happen during file initialization with the try/except. This will reduce (a small) amount of overhead of re-importing. We can switch ImportError to NameError within the function to keep the same functionality